src/chrome/content/rules/fir.im.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	Mismatch：
+		firicon.fir.im
+		pkg3.fir.im
+
+-->
+
+<ruleset name="fir.im">
+
+	<target host="fir.im" />
+	<target host="account.fir.im" />
+	<target host="api.fir.im" />
+	<target host="blog.fir.im" />
+	<target host="club.fir.im" />
```

Comment:
For this host, I'm getting image requests timeouts on the HTTPS endpoint - the HTTP endpoint loads quickly.